### DISCUSSION TITLE
Not able to Access my Learning Section

### DISCUSSION THREAD

**Discussion by TvaritPatelSD** [github/india#403] [General]
Hey Guys i will not able to access my Learning section,it will show me the error as i mentioned in the image since last 5 days, and i have already drooped mail to Support Team but no one replied to me Please Let me Know if Anyone is Facing the same issue as i'm currently facing, Thank you.

https://user-images.githubusercontent.com/85013937/136832274-c71712d1-4ae2-41c7-b83c-b91185d57c81.jpeg

**Comment by AbhishekMondal1** [comment]
Did you tried to logged out and log in again?

**Reply by TvaritPatelSD** [reply]
Oh Ya Thanks it Works like Charm, but still not able to access any section like Self Learning , Projects not even Industry Session section i already Filed all the Required Fields then why i'm not able to Access these Sections ?

**Reply by AbhishekMondal1** [reply]
Does all of your profile links showing filled out after refreshing the page in orientation and the Orientation tab is show green or not?

**Reply by TvaritPatelSD** [reply]
Thanks for Your Replay, Yes All Profile Links are Showing Filled out but the Orientation Session tab is not Green.

**Reply by AbhishekMondal1** [reply]
Do one thing. Press **F12** and go to developer tabs. Go to the **Network** tab and see the network request to **GetDetails?=**. Click it and then press the **Preview** tab. Send the response if there is orientation showing completed or incomplete, if there is attachmentLink or not
![details-info](https://user-images.githubusercontent.com/71382408/136963110-d3de465c-ef9f-4406-9daa-194aa312a093.png)
.

**Reply by TheSuranaverse** [reply]
A group for all to get notified for any `Official FRT Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Currently, we are helping all the `Future Ready Talent` Interns clear their doubts & queries on a personal basis either on chat or voice, whichever they prefer. We will also be assisting them on Projects & help them complete this internship 🤗

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Comment by TvaritPatelSD** [comment]
Here is the Response 
![Capture](https://user-images.githubusercontent.com/85013937/136986873-0bbb96ac-e9f5-4d53-8468-728ff4aa7f2c.PNG)

**Reply by AbhishekMondal1** [reply]
I can't see anything wrong in ur profile. All the links are submitted. But I think the /collections is not needed at the microsoft link, remove it and submit again.

**Reply by TvaritPatelSD** [reply]
Great it's Working Fine now Thank you Very Much For Your Help.

**Reply by TheSuranaverse** [reply]
A group for all to get notified for any `Official FRT Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Currently, we are helping all the `Future Ready Talent` Interns clear their doubts & queries on a personal basis either on chat or voice, whichever they prefer. We will also be assisting them on Projects & help them complete this internship 🤗

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Reply by TvaritPatelSD** [reply]
@AbhishekMondal1 After Completing the first Topic of Microsoft Azure it is still shows me that it is In Progress  what can i do?
![image](https://user-images.githubusercontent.com/85013937/137350564-fcab1c52-19fb-46b3-a30e-3e34d2a001e6.png)
![image](https://user-images.githubusercontent.com/85013937/137350835-eb6438c7-e32f-44b3-9f4b-c4966750c95a.png)

**Reply by AbhishekMondal1** [reply]
Its showing same to all the interns. They told every week it will be get updated. May be tomorrow it will be updated to new status.

**Reply by TvaritPatelSD** [reply]
okay Thanks.

**Reply by ananya0504** [reply]
Hey @TvaritPatelSD 
I just signed up for the October registrations. What is the next step that I should take?
